### PR TITLE
feat: add support for Boom errors

### PR DESCRIPTION
**Pull request by dominiklessel**
This PR adds Boom error support to createHttpHandler.

### CODE REVIEW

File: src/http/handler.ts [hunk 1]
```diff
@@ -94,6 +95,18 @@ export const createHttpHandler =
             body: error.message,
           };
         }
+        if (error instanceof Boom.Boom) {
```
Comment (hoegertn) [inline src/http/handler.ts]: If you check this by looking for `error.isBoom` you could skip the explicit dependency on Boom. So other people would not make their bundle bigger for no reason.

**Comment by hoegertn** [comment]
I like the idea but would love to skip the dependency

**Comment by dominiklessel** [comment]
7-format2020

**Comment by dominiklessel** [comment]
BTW, once merged, I'd really appreciate if you could bump cdk-serverless's deps as well 😃

**Comment by hoegertn** [comment]
> BTW, once merged, I'd really appreciate if you could bump cdk-serverless's deps as well 😃

Will do. Happy to get feedback on cdk-serverless...

**Pull request merged**